### DISCUSSION TITLE
GHelper Greyed Out Drivers in Update tab

### DISCUSSION THREAD

**Discussion by bynureal** [seerge/g-helper#1283] [General]
What does it mean if multiple drivers are greyed out inside of the update tab in GHelper? I don't know whether or not they need to be update because usually they are red when update is needed, and if I try downloading the driver on one of the greyed out ones it'll download but stay greyed out in the update tab. Is this normal?

**Comment by seerge** [comment]
It's when app doesn't know your local version of that "update" (it's mostly software components)

**Reply by brambedkar59** [reply]
Some of the drivers are also greyed out even though they are up-to-date.

**Reply by seerge** [reply]
For some drivers it’s also not possible to get local version
